src/components/extended-action-fab/extended-action-fab.tsx

```diff
@@ -169,12 +169,12 @@ const dynamicStyles = new DynamicStyleSheet({
     alignItems: 'center',
     padding: theme.spacing.s,
     width: '100%',
+    zIndex: 10,
```

Comment:
Before merging this, we need to check and make sure that this doesn't break anything on Android